Source/AppKit additions/SVGKImageRep.h

```diff
@@ -0,0 +1,38 @@
+//
```

Comment:
This file is original from [MaddTheSane](https://github.com/MaddTheSane/SVGKit/blob/master/Source/AppKit%20additions/SVGKImageRep.h).